docs/UserGuide.md

```diff
@@ -231,11 +227,13 @@ Expected outcome:
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add {module code} {modular credits}` <br> e.g. `add CS2100 4`
-**Edit** | `edit {index} [c/module description] [cr/modular credits] [t/tags]` <br> e.g. `edit 1 c/CS2103T t/core`
-**Delete** | `delete {module}`<br> e.g. `delete CS2103T`
-**List** | `list`
+**Help** | `help`
+**Add** | `add [c/module code] [cr/modular credits]` <br> e.g. `add c/CS2100 cr/4`
```

Comment:
I think square brackets are only for optional arguments? Actually are we following the original UG syntax format ahhh? I'm not too sure

Reply:
Oh yeah forgot about this. Thanks for catching it! 👍